src/server.js

```diff
@@ -132,6 +133,12 @@ function jsonExporter(req, res, next) {
             id,
             title,
             description,
+            ...(preview_image_link && {
+              preview_image_link: preview_image_link['@id'].replace(
```

Comment:
Can we add another `jq` pipe? it should look like this (I think)

```
    .then((content) => {
      return run(
        `(.. | .preview_image_link? | arrays | .[]."@id") |= sub("${config.settings.apiPath}";"")`,
        content,
        {
          input: 'json',
          output: 'json',
        },
      );
    })
```

then we can remove the ugly replace (and it will work for nested use cases too)